logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/GetNotificationsUseCase.kt

```diff
@@ -118,20 +116,65 @@ class GetNotificationsUseCaseImpl(
             .distinctUntilChanged()
     }
 
+    private suspend fun observeMessagesList(conversation: Conversation) =
+        if (conversation.lastNotificationDate == null) {
+            // that is a new conversation, lets just fetch last 100 messages for it
+            messageRepository.getMessagesByConversationIdAndVisibility(
+                conversation.id,
+                100,
```

Comment:
You'll probably have to extract this to a variable to avoid the "MagicNumber" detekt warning